scripts/pipeline/populate_per_genome_database.pl

```diff
@@ -0,0 +1,291 @@
+#!/usr/bin/env perl
+# See the NOTICE file distributed with this work for additional information
+# regarding copyright ownership.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+use warnings;
+use strict;
+
+=head1 NAME
+
+populate_per_genome_database.pl
+
+=head1 DESCRIPTION
+
+This script copies from pipeline db to an empty per-species compara db.
+
+This script only copies the data from the tables corresponding to the provided
+compara_db for the provided list of tables.
+
+=head1 SYNOPSIS
+
+perl populate_per_genome_database.pl --help
+
+perl populate_per_genome_database.pl
+    --pipeline_db pipeline_db_url
+    --compara_db per_species_db_url
+    --tables "genome_db,method_link_species_set_id..."
+
+=head1 REQUIREMENTS
+
+This script uses mysql, mysqldump and mysqlimport programs.
+It requires at least version 4.1.12 of mysqldump as it uses
+the --insert-ignore option.
```

Comment:
Based on our experience with `mysqlimport` in the main release homology, I would suggest to use replace in case the job fails, so partial rows are copied again correctly (you are not doing the copy in a transaction manner so there is no rollback, right?).